server/errorHandler.ts

```diff
@@ -20,9 +20,39 @@ export default function createErrorHandler(production: boolean) {
       if (err.status === 403 && (!err.external || err.response?.body?.accessErrors)) {
         res.status(403)
 
+        const userHeaderService = 'You do not have permission to view this service'
+        const userHeaderPage = 'You do not have permission to view this page'
+        const returnedError = err.response?.body?.message
+        let userMessage
+        let userHeader
+
+        if (returnedError === 'could not map service provider user to access scope') {
```

Comment:
@JakeGoodwin22 is there any way to externalise these messages to an error-mapping.properties (error -> errormessage) or something similar. So, in the future , adding or amending the messages would be easier. This is just a suggestion, please consider this if it is feasible.

Reply:
There is an error messages file, if that's what you mean. `server/utils/errorMessages.ts `

Reply:
Happy to move these to the errorMessages file, thanks!